docs/components/board/orangepi.md

```diff
@@ -9,12 +9,13 @@ tags: ["board", "components", "orangepi"]
 # SMEs: Olivia Miller
 ---
 
-Configure an `orangepi` board to integrate an [Orange Pi Zero2](http://www.orangepi.org/html/hardWare/computerAndMicrocontrollers/details/Orange-Pi-Zero-2.html) or [OrangePi 3 LTS](http://www.orangepi.org/html/hardWare/computerAndMicrocontrollers/details/orange-pi-3-LTS.html) into your machine.
+Configure an `orangepi` board to integrate the GPIO pins of an [Orange Pi Zero2](http://www.orangepi.org/html/hardWare/computerAndMicrocontrollers/details/Orange-Pi-Zero-2.html) or [OrangePi 3 LTS](http://www.orangepi.org/html/hardWare/computerAndMicrocontrollers/details/orange-pi-3-LTS.html) into your machine.
 
-If you have an Orange Pi Zero2, follow [this guide to flash your board](/get-started/installation/prepare/orange-pi-zero2/).
-Follow the {{< glossary_tooltip term_id="setup" text="setup instructions" >}} to install `viam-server` on your board and connect to the Viam app.
+If you have an Orange Pi Zero2, first follow [this guide to flash your board](/get-started/installation/prepare/orange-pi-zero2/).
+If you have an Orange Pi 3 LTS, follow [this guide](/get-started/installation/prepare/orange-pi-3-lts/) instead.
```

Comment:
Nit: naming links usefully, to avoid `[this guide]` and `[that other guide]`. Also calling out the choice a little more prominantly:

```suggestion
First, follow the installation guide for your specific Orange Pi board:

- For an Orange Pi Zero2: follow the [Orange Pi Zero2 installation guide](/get-started/installation/prepare/orange-pi-zero2/).
- For an Orange Pi 3 LTS, follow the [Orange Pi 3 LTS installation guide](/get-started/installation/prepare/orange-pi-3-lts/).
```

This approach duplicates board names in the description and also the link, which is a bit cumbersome I admit. I prefer even this explicitness, but if you want to remove dupe names, might there be a way to use the name in the link title only? Up to you.

Reply:
I'm fine with this!